src/Atmos/Model/turbulence.jl

```diff
@@ -95,18 +95,18 @@ end
 
 vars_gradient(::ConstantViscosityWithDivergence,FT) = @vars()
 vars_diffusive(::ConstantViscosityWithDivergence, FT) =
-  @vars(S::SHermitianCompact{3,FT,6})
+  @vars(∇u::SMatrix{3,3,FT,9}) #@vars(S::SHermitianCompact{3,FT,6})
```

Comment:
Please remove commented lines of code once you're certain the change is correct.

Reply:
Why do we need to modify these? Does the Courant number calculation require `∇u` instead of `S`?

Reply:
@simonbyrne actually the courant number requires only S, however depending on the turbulence model used only S or ∇u is saved. The way I had previously written it was to have them both saved but since this is much more memory intensive I opted to have all models save ∇u and calculate S whenever necessary as per Jeremy's suggestion

Reply:
File reset to master.